src/Perspex.SceneGraph/RelativeRect.cs

```diff
@@ -191,14 +191,14 @@ public static RelativeRect Parse(string s, CultureInfo culture)
                     parts[2] = parts[2].TrimEnd('%');
                     parts[3] = parts[3].TrimEnd('%');
                     unit = RelativeUnit.Relative;
-                    scale = 0.01;
+                    scale = 0.01m;
                 }
 
                 return new RelativeRect(
-                    double.Parse(parts[0], culture) * scale,
-                    double.Parse(parts[1], culture) * scale,
-                    double.Parse(parts[2], culture) * scale,
-                    double.Parse(parts[3], culture) * scale,
+                    (double)(decimal.Parse(parts[0], culture) * scale),
```

Comment:
It's so the test passes right? I'd suggest fixing the test to only check x significant digits, or use a value that has a strict FP representation rather than doing this.